model/src/main/kotlin/config/Excludes.kt

```diff
@@ -113,14 +121,13 @@ data class Excludes(
      * True if the [scope] is excluded in [project] by this [Excludes] configuration.
      */
     fun isScopeExcluded(scope: Scope, project: Project, ortResult: OrtResult) =
-        scopes.any { it.matches(scope.name) }
-                || findProjectExclude(project, ortResult)?.scopes?.any { it.matches(scope.name) } ?: false
+        findScopeExcludes(scope, project, ortResult).isNotEmpty()
 
     /**
      * Map the project excludes by the identifiers of the provided [projects].
      */
     fun projectExcludesById(projects: Set<Project>, ortResult: OrtResult) =
-        projects.associate { Pair(it.id, findProjectExclude(it, ortResult)) }
+        projects.associate { Pair(it.id, findProjectExcludes(it, ortResult).singleOrNull()) }
```

Comment:
Wouldn't the correct fix be to not use `singleOrNull` here? Or do you plan to fix that separately?